### ISSUE TITLE
How i can send LOCATION longitude latitude by URL to server

### ISSUE THREAD

**Issue by yesamine** [transistorsoft/cordova-background-geolocation-lt#482]
hi @christocracy 
i try to send location latitude and longitude by url post or get method but i cant i recive ping log with static params in text url (like GET method)
i use php in my server i recive :
$_GET[lng] => test
$_GET[lat] => <%= latitude %>
and the other params not work 

my code i know need to change :)
```
bgGeo.configure({
        // Geolocation config
        desiredAccuracy: 0,
        distanceFilter: 10,
        stationaryRadius: 25,
        // Activity Recognition config
        activityRecognitionInterval: 10000,
        stopTimeout: 5,
        // Application config
        debug: true,  // <-- Debug sounds & notifications.
        stopOnTerminate: false,
        startOnBoot: true,
        forceReloadOn:true,
        // HTTP / SQLite config
        url: "http://view.xtrem360.com/api/set.php?start&lng=test&lat=<%= latitude %>",
        httpRootProperty: '.',
        locationTemplate: '{ "latitude": <%= latitude %>, "longitude": <%= longitude %> }', // not work
        method: "POST",
        autoSync: true,
        maxDaysToPersist: 3,
        headers: {  // <-- Optional HTTP headers
            "X-FOO": "bar"
        },
        params: {   // <-- Optional HTTP params
            "auth_token": "maybe_your_server_authenticates_via_token_YES?",
            "latitude": '<%= longitude %>', // not work
        }
    }, function(state) {
        // This callback is executed when the plugin is ready to use.
        console.log("BackgroundGeolocation ready: ", state);
        if (!state.enabled) {
            bgGeo.start();
        }
    });
```

im on android i try to get localisation of a client and send notification to this client if is in zon

**Comment by christocracy** [comment]
https://github.com/transistorsoft/cordova-background-geolocation-lt/issues/2

**Comment by yesamine** [comment]
ok i look in #2 but i not understand where the params of location sets and way use localStorage ?? are u store information in it ??
plz give me more information for good understanding

thanks @christocracy

**Comment by christocracy** [comment]
You find the data in php $_GET or $_POST.

The data is available as shown in #2.

The plugin has nothing to do with LocalStorage.
